power/src/tests.rs

```diff
@@ -63,36 +63,55 @@ fn one_sample_t_test() {
     test_interface(&join(&extra), 0.967);
     let extra = json!({"tail": "2", "analysis": "power"});
     test_interface(&join(&extra), 0.934);
+    let extra = json!({"tail": "1", "analysis": "es"});
+    test_interface(&join(&extra), 0.472);
+    let extra = json!({"tail": "2", "analysis": "es"});
+    test_interface(&join(&extra), 0.520);
+    let extra = json!({"tail": "1", "analysis": "n"});
+    test_interface(&join(&extra), 45.0);
+    let extra = json!({"tail": "2", "analysis": "n"});
+    test_interface(&join(&extra), 54.0);
 }
 
 #[test]
-fn deviation_from_zero_multiple_regression() {
-    let join = with_rest("deviationFromZeroMultipleRegression");
-    let f_squared = ES.sqrt();
-    let extra = json!({"nPredictors": "2", "es": f_squared, "analysis": "alpha"});
-    test_interface(&join(&extra), 0.006);
-    let extra = json!({"nPredictors": "2", "es": f_squared, "analysis": "power"});
-    test_interface(&join(&extra), 0.994);
-    let extra = json!({"nPredictors": "2", "es": f_squared, "analysis": "es"});
-    test_interface(&join(&extra), 0.574);
-    let extra = json!({"nPredictors": "2", "es": f_squared, "analysis": "n"});
-    test_interface(&join(&extra), 34.0);
+fn independent_samples_t_test() {
+    // G*Power only gives 0.392 if you put sample size group 1 and 2 both on n=50.
+    // pwr.t.test(n=50, d=0.5, sig.level=NULL, power=0.95, type="two.sample", alternative="two.sided")
+    let join = with_rest("independentSamplesTTest");
+    let extra = json!({"tail": "2", "analysis": "alpha"});
+    test_interface(&join(&extra), 0.398);
+    let extra = json!({"tail": "1", "analysis": "n"});
+    test_interface(&join(&extra), 88.0);
 }
 
 #[test]
 fn goodness_of_fit_chisq() {
     let df = "5";
     let join = with_rest("goodnessOfFitChisqTest");
-    let extra = json!({"analysis": "alpha", "df": df});
+    let extra = json!({"df": df, "analysis": "alpha"});
     test_interface(&join(&extra), 0.254);
-    let extra = json!({"analysis": "alpha", "df": df, "es": 0.628});
+    let extra = json!({"df": df, "es": 0.628, "analysis": "alpha"});
     test_interface(&join(&extra), 0.051);
-    let extra = json!({"analysis": "power", "df": df});
+    let extra = json!({"df": df, "analysis": "power"});
     test_interface(&join(&extra), 0.788);
-    let extra = json!({"analysis": "es", "df": df});
+    let extra = json!({"df": df, "analysis": "es"});
     test_interface(&join(&extra), 0.629);
-    let extra = json!({"analysis": "n", "df": df});
-    test_interface(&join(&extra), 79.0);
+    let extra = json!({"df": df, "analysis": "n"});
```

Comment:
I put these towards the end of the line, because when you skim through the tests is REALLY easy to see it we are testing for:

- alpha
- power
- es
- n

Reply:
And I made this consistent across tests.